src/mono/wasm/wasm.proj

```diff
@@ -148,7 +148,7 @@
     </ItemGroup>
     <PropertyGroup>
       <PInvokeTableFile>$(ArtifactsObjDir)wasm/pinvoke-table.h</PInvokeTableFile>
-      <CMakeConfigurationEmccFlags Condition="'$(Configuration)' == 'Debug'">-g;-Os;-s;-DDEBUG=1</CMakeConfigurationEmccFlags>
+      <CMakeConfigurationEmccFlags Condition="'$(Configuration)' == 'Debug'">-g -Os -s -DDEBUG=1</CMakeConfigurationEmccFlags>
```

Comment:
@radekdoulik do you remember why we switched to `;` ? I know it's list separator, but I don't recall what was broken. Possibly something on windows emcc

Reply:
The problem on windows was that it was added as single option to the emcc. I think it might be also different for different cmake versions.

set_target_properties might be different. I will try it on my windows machine